Sources/PostgresNIO/Data/PostgresData.swift

```diff
@@ -70,6 +74,10 @@ public struct PostgresData: CustomStringConvertible, CustomDebugStringConvertibl
             description = self.array(of: String.self)?.description
         case .jsonbArray:
             description = self.array?.description
+        case .int4RangeArray:
+            description = try? [PostgresRange<Int32>](from: &value, type: type, format: formatCode, context: .default).description
+        case .int8RangeArray:
+            description = try? [PostgresRange<Int64>](from: &value, type: type, format: formatCode, context: .default).description
```

Comment:
since we don't want to support this in PostgresData anymore, is this still needed?

Reply:
Removed here, along with two protocol conformances that enabled it: https://github.com/vapor/postgres-nio/pull/330/commits/24c92570c4c7537923ee723b9fb0efd15c7dbdc1